src/shared/components/palette/address/write-address-field.component.ts

```diff
@@ -129,8 +129,8 @@ export class WriteAddressFieldComponent extends AbstractFieldWriteComponent impl
     }
   }
 
-  createId(elementId: string): string {
-    return this.id() + '_' + elementId ;
+  buildIdPrefix(elementId: string): string {
+    return this.idPrefix + '_' + elementId ;
   }
 
```

Comment:
I have moved `createId `into the abstract class as it is used by multiple components and created a new `buildIdPrefix ` method for dynamic inner components in the write-address-field